examples/basics/visuals/line.py

```diff
@@ -7,6 +7,9 @@
 """
 Demonstration of various features of Line visual.
 """
+
+from __future__ import division
```

Comment:
Ah I wasn't sure if you supported Python 2.6. Still, I prefer to do this to be explicit to make / behave the same on 2.x and 3.x